subprojects/docs/src/docs/userguide/more_about_tasks.adoc

```diff
@@ -481,169 +582,142 @@ For this example, we’re using a custom class to represent the model data. Howe
 +
 The directory where the generated files go. As with input files, there are several annotations for output files and directories. A property representing a single directory requires `@OutputDirectory`. You’ll learn about the others soon.
 
-These annotated properties mean that Gradle will skip the task if none of the source files, template engine, model data or generated files have changed since the previous time Gradle executed the task. This will often save a significant amount of time. You can learn how Gradle detects <<sec:how_does_it_work,changes later>>.
+These annotated properties mean that Gradle will skip the task if none of the source files, template engine, model data or generated files have changed since the previous time Gradle executed the task. This will often save a significant amount of time. You can learn how Gradle detects <<#sec:how_does_it_work,changes later>>.
 
-This example is particularly interesting because it works with collections of source files. What happens if only one source file changes? Does the task process all the source files again or just the modified one? That depends on the task implementation. If the latter, then the task itself is incremental, but that’s a different feature to the one we’re discussing here. Gradle does help task implementers with this via its <<incremental_tasks,incremental task inputs>> feature.
+This example is particularly interesting because it works with collections of source files. What happens if only one source file changes? Does the task process all the source files again or just the modified one? That depends on the task implementation. If the latter, then the task itself is incremental, but that’s a different feature to the one we’re discussing here. Gradle does help task implementers with this via its <<custom_tasks.adoc#incremental_tasks,incremental task inputs>> feature.
 
 Now that you have seen some of the input and output annotations in practice, let’s take a look at all the annotations available to you and when you should use them. The table below lists the available annotations and the corresponding property type you can use with each one.
 
-.Incremental build property type annotations
-[cols="a,a,a", options="header"]
-|===
-| Annotation
-| Expected property type
-| Description
 
-| `@api:org.gradle.api.tasks.Input[]`
-| Any serializable type
-| A simple input value
+===== Incremental build property type annotations
 
-| `@api:org.gradle.api.tasks.InputFile[]`
-| `File`*
-| A single input file (not directory)
+`@link:{javadocPath}/org/gradle/api/tasks/Input.html[Input]` — Expected property type: Any `Serializable` type::
+A simple input value
 
-| `@api:org.gradle.api.tasks.InputDirectory[]`
-| `File`*
-| A single input directory (not file)
+`@link:{javadocPath}/org/gradle/api/tasks/InputFile.html[InputFile]` — Expected property type: `File`*::
```

Comment:
The footnote symbols here don't match the old document.  e.g., we're using `*`, `†` and `‡` in the "table", but we only have footnotes for `†` and `‡`. 

It might be clearer if we do this a little differently.  e.g., instead of saying `File` with a footnote, we could say _file-like_ and then define what that means.

Reply:
I'll just use the old symbols for now. Perhaps we can make different content changes in a separate PR.